### ISSUE TITLE
Redshift features not supported yet

### ISSUE THREAD

**Issue by tunetheweb** [sqlfluff/sqlfluff#1642]
Copied from: https://github.com/sqlfluff/sqlfluff/issues/612#issuecomment-848086315

Reporting a few more examples of Redshift-specific functionality:

The COPY command available in Redshift does not exist in PostgreSQL, which makes it impossible to sqlfluff lint or sqlfluff fix any COPY query, e.g.:
In test.sql:

```sql
COPY catdemo
FROM 's3://awssampledbuswest2/tickit/category_pipe.txt'
IAM_ROLE 'arn:aws:iam::<aws-account-id>:role/<role-name>'
REGION 'us-west-2';
```

❯ sqlfluff lint test.sql
== [test.sql] FAIL
L:   1 | P:   1 |  PRS | Found unparsable section: "COPY catdemo\nFROM
                       | 's3://awssampledbuswes..."

A similar scenario occurs with the CREATE EXTERNAL TABLE Redshift command.

Redshift INSERT, DELETE, and UPDATE statements do not support WITH clauses as PostgreSQL does. This opens the door to having a Redshift-specific rule to catch this.

**Comment by farfromunique** [comment]
Redshift dialect support was added in 0.7.0, it seems. Should this issue be closed?

**Comment by tunetheweb** [comment]
There were a few items listed in #612 before Redshift was supported. The ones above were not in that initial support (when I closed #612 as we no do support Redshift as you say). So copied to this new issue. Not sure if those specific examples are supported now, and they probably should be individual, more detailed descriptions to be honest.

Let's close and get people to open new issues with more detail if they need those functionalist and they don't yet exist.
